src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -58,186 +65,163 @@ public static SidecarInfo createSidecarInfo(TestInputs input) {
   @TestFactory
   Stream<DynamicTest> testCombinations() {
 
-    final String vscodeVersion = String.valueOf(new SidecarInfo().vsCode());
 
     List<TestInputs> inputs = List.of(
         // Linux
         new TestInputs(
             "Linux OS info with VS Code",
-            "Linux",
-            "22.0413",
+            "linux",
+             "aarch64",
+            "20.1.2",
             OperatingSystemType.Linux,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             null,
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION)
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v20.1.2 (linux/aarch64)"
         ),
         new TestInputs(
             "Linux OS info with VS Code and version prefix",
-            "Linux",
-            "22.0413",
+            "linux",
+            "x86_64)",
+            "20.1.2",
             OperatingSystemType.Linux,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             "v",
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION)
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (linux/aarch64)"
         ),
         new TestInputs(
             "Linux OS info without VS Code",
-            "Linux",
-            "22.0413",
+            "linux",
+            "x86_64",
+            "20.1.2",
             OperatingSystemType.Linux,
             null,
             null,
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION)
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (linux/aarch64)"
         ),
         // Mac OS
         new TestInputs(
             "Mac OS info with VS Code",
-            "Mac OS X",
-            "13.1",
+            "mac os x",
+            "x86_64",
+            "20.1.2",
             OperatingSystemType.MacOS,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
             "",
-            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Mac OS X/x86_64)".formatted(
-                vscodeVersion,
-                SidecarInfo.VERSION)
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (macosx/x86_64)"
```

Comment:
This should be:
```suggestion
            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v5.40.3 (mac/x86_64)"
```
if we really are using `OperatingSystemType` literals.

Reply:
And this is because of 
```
    if (lowerName.contains("mac") || lowerName.contains("darwin")) {
      return OperatingSystemType.MacOS;
    }
```
in `OperatingSystemType` I imagine. The problem is sidecarinfo seems to return the uppercase type:

```
Expected :Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/v20.1.2 (linux/aarch64)
Actual   :Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/vunset (Linux/aarch64)
```

working on it.